.gitignore

```diff
@@ -131,3 +131,6 @@ sponge-theme/
 
 # Source code created by pip
 src/
+
+# compiled sponge-javadoc extension
+etc/sponge-javadoc/sponge-javadoc.pyc
```

Comment:
You can just add `*.pyc` directly, they should never be uploaded to the docs repo